### PR TITLE
Add service to doctor

### PR DESCRIPTION
**Pull request by IrinaEndeladze**

### CODE REVIEW

File: components/tabs/AboutDoctorTab.tsx [hunk 1]
```diff
@@ -93,6 +93,8 @@ export default function AboutDoctorTab({
 				if (error.response) console.log(error.response);
 			});
 	};
+
+	useEffect(() => {}, [doctor]);
```
Comment (NikitoriusMinsk) [inline components/tabs/AboutDoctorTab.tsx:97]: remove empty useEffect

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]

**Comment by NikitoriusMinsk** [comment]
Check merge conficts

File: pages/doctors/doctors_detailed/index.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ export default function DoctorsDetailed() {
 	const [serviceAddModal, serServiceAddModal] = useState(false);
 
 	var { data, refetch, isLoading, isError, error, status } = useQuery(
-		['key', 'doctorDetailed'],
+		[id, 'FrelnacerAndClinicDoctors'],
```
Comment (NikitoriusMinsk) [inline pages/doctors/doctors_detailed/index.tsx:59]: change id to doctor_id to avoid conflicting cache keys

**Review by NikitoriusMinsk — CHANGES_REQUESTED** [review]

File: pages/doctors/doctors_detailed/index.tsx (hunk 2 above)
Comment (IrinaEndeladze) [reply pages/doctors/doctors_detailed/index.tsx:59]: this  id which i use, is doctor's id? you mean to write like this  "doctor_id" ?

Comment (NikitoriusMinsk) [reply pages/doctors/doctors_detailed/index.tsx:59]: `doctor_${id}`

**Pull request merged**